demo/src/App.vue

```diff
@@ -220,6 +236,44 @@ onMounted(() => {
   plugin.draw('root');
   updateComponentsIds();
 });
+
+/**
+ *
+ */
+function addComponent() {
+  const id = plugin.data.addComponent(plugin.data.__laptopDefinition);
+
+  // As of now, we need to draw the component before repositioning it.
+  // That is because we need the component drawOption property to be defined.
+  plugin.draw('root');
+
+  plugin.repositionComponent(id);
+  plugin.draw('root');
+}
+
+/**
+ *
+ */
+function gherkinTestsToConsole() {
+  console.log('-------------------- Start log position');
+  const log = plugin.data.components
+    .map(({ id }) => document.querySelector(`g #${id}`))
+    .map((e) => {
+      if (e.attributes.x === undefined || e.attributes.y === undefined) {
+        return `'${e.id}' does not have valid position`;
+      }
+
+      return `And I expect "#${e.id}" to be at position`
+          + ` ${e.attributes.x.nodeValue},${e.attributes.y.nodeValue}\n`;
+    })
+    .join('');
+
+  // Using only one console.log allows to copy/paste easily from the console.
+  // Using multiple console.log calls would add unwanted empty lines.
+  console.log(log);
```

Comment:
remove this.

Reply:
Why ? Using multiple console.log make it hard to copy/paste tests (at least in Chrome).
The goal of this feature is to speed up development, not to slow it down.

Reply:
done